docs/blog/2018-12-15-develop-and-deploy-your-own-react-monorepo-app/index.md

```diff
@@ -0,0 +1,598 @@
+
+This tutorial deals with loads of small changes which make for plenty of opportunities for things to go wrong. I’ve tried making myself as clear as possible to prevent this but if you do come across issues then leave a descriptive commentI’ll do my best to respond, edit and improve this article. You can also find the [finished repo here](https://github.com/andrico1234/lerna-monorepo-finisher). There’s no shame in copying my code if you’re stuck.
+
+I’m also assuming fundamental web dev knowledge including experience with React, Git and the command line.
+
+### Part One: Justine (Development)
+
+Let’s begin! Navigate to an empty folder in the terminal and clone the repo by running the following command
+
+`git clone git@github.com:andrico1234/lerna-monorepo-starter.git`
+
+and navigate into the the `lerna-monorepo-starter` directory.
+
+Go to your GitHub account and create a new repo, this’ll be the repo that you push the site to so that Travis can run the pipeline. For those who need a refresher on how to [create a new repo](https://help.github.com/articles/creating-a-new-repository/). You may have to run `git remote remove origin` so you can point the code to your newly created repo.
+
+Run the code that GitHub outputs when creating a new repo:
+
+```
+git remote add origin git@github.com:andrico1234/lerna-monorepo-runthrough.git
+git push -u origin master
+```
+
+Commit and push regularly for best results.
+
+Navigate inside of the `lerna-monorepo-starter` directory and install the packages by running `yarn`. Once your packages have installed, run `yarn develop` to run the application on localhost:8000.
+
+Open Chrome, and navigate to localhost:8000. You’ll find a simple application with a mock blog and a mock shop. Imagine this is a fleshed out application, we can make the assumption that the underlying logic behind the blog and the shop are different and that there are similarities with the styling and components.
+
+Go back to your IDE and you’ll see that we’re not pulling data from an external source. The data is being served locally via a couple of json files in the `data` directory. When we run the `develop` command Gatsby pulls the data from these json files and creates pages for them. You can take a look at how this being done by peeking inside of the `gatsby-node.js` file. This is because Gatsby builds the code and outputs a static site.
+
+So imagine our application is starting to grow, the shop and the blog start intertwining with each other. You may not notice the lack of separation until its too late and your code is tangled tighter than the ivy across the side of your grandma’s cottage.
+
+Let’s start splitting up our application. I’ll only be touching the surface of what Lerna is capable of, so I’d recommend checking out their repo to learn more. Install Lerna with `yarn global add lerna`. We need to create a file in the root directory called `lerna.json`. Add in the following:
+
+```
```

Comment:
We have some brand new ways of making file names and code types clear

file name using

```shell
```jsx:title=gatsby-config.js
```
at the top of the code snippet. 


type tabs
#10515

Reply:
I'd recommend this, though it's still optional in the blog. We are requiring it in docs now

Reply:
thanks, i'll get that in!